src/ExportEdiJobs/ExportEdiJobDetailsActionMenu/ExportEdiJobDetailsActionMenu.js

```diff
@@ -3,25 +3,47 @@ import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
 import { pick } from 'lodash';
 
-import { IfPermission } from '@folio/stripes/core';
+import {
+  IfPermission,
+  useOkapiKy,
+} from '@folio/stripes/core';
 import {
   Button,
   Icon,
   MenuSection,
 } from '@folio/stripes/components';
-import { useShowCallout } from '@folio/stripes-acq-components';
+import {
+  downloadBase64,
+  useShowCallout,
+} from '@folio/stripes-acq-components';
 
+import {
+  EXPORT_JOBS_API,
+  EXPORT_JOB_STATUSES,
+} from '../../common/constants';
 import { useExportJobScheduler } from '../../common/hooks';
 import { useNavigation } from '../../hooks';
 
 export const ExportEdiJobDetailsActionMenu = ({
   exportJob,
+  isRerunDisabled,
   onToggle,
   refetchJobs,
 }) => {
+  const ky = useOkapiKy();
+  const showCallout = useShowCallout();
   const { scheduleExportJob } = useExportJobScheduler();
   const { navigateToEdiJobDetails } = useNavigation();
-  const showCallout = useShowCallout();
+
+  const {
+    name,
+    fileNames,
+    status,
+  } = exportJob;
+
+  const canBeResendOrDownload = (
+    status === EXPORT_JOB_STATUSES.successful || (status === EXPORT_JOB_STATUSES.failed && fileNames?.length)
```

Comment:
does it mean just fileNames?.length > 0?

Reply:
It looks like it, thanks)